src/tests/ftest/launch.py

```diff
@@ -1196,12 +1196,12 @@ def _get_build_environment(list_tests):
 
         except ValueError as error:
             if not list_tests:
-                raise LaunchException("Error setting test environment") from error
+                raise LaunchException("Error setting test environment:", str(error)) from error
```

Comment:
Curious, why? The `error` should be in the traceback since `from error` is used.

Reply:
I was having a hard time debugging an issue and didn't see the previous error in the stack trace.

Reply:
Ah, maybe because we don't print the traceback when catching the exception:
https://github.com/daos-stack/daos/blob/622f0652b8377fe58454473493759733d88fb926/src/tests/ftest/launch.py#L941-L945
I think this is okay for now, but @phender FYI